packages/azpipelines/BuildTasks/TriggerApexTestTask/task.json

```
@@ -64,6 +64,23 @@
             "helpMarkDown": "Select an option if the tests are to be run synchronously",
             "required": false
         },
+        {
+            "name": "isValidateCoverage",
+            "type": "boolean",
+            "label": "Validate code coverage of individual test classes",
+            "defaultValue": false,
+            "helpMarkDown": "When enabled, verifies whether indvidual test classes meet minimum code coverage requirement",
+            "required": false
+        },
+        {
+            "name": "coverageThreshold",
+            "type": "string",
+            "label": "Minimum percentage coverage required per test class",
+            "defaultValue": "75",
+            "helpMarkDown": "Minimum coverage required per test class, in order for the task to succeed",
+            "required":true,
```

```suggestion
            "required":false,
```
Lets not make this a required field

---

But what if someone leaves out the field?

---

defaultValue will help,  if we make it required, i believe it will create an issue when used in YAML mode as it doesn't respect visiiblity